app/services/valid_test_data_generators/separation_shared_data.rb

```diff
@@ -1,66 +0,0 @@
-# frozen_string_literal: true
-
-module ValidTestDataGenerators
-  class SeparationSharedData < ECFLeadProviderPopulater
```

Comment:
can we change this instead to sandbox, we might need this later if we ever rerun that data?